### ISSUE TITLE
In 2.1, accumulo shell exhibits unexpected copy/paste behavior

### ISSUE THREAD

**Issue by keith-ratcliffe** [apache/accumulo#3170]
**Describe the bug**

In prior versions of Accumulo, the user could copy and paste a newline-separated list of commands into the shell and have them be executed in sequence. In 2.1, this seems to be no longer possible. When the user attempts to execute the pasted commands, it seems that nothing is actually executed. A workaround is to place the commands in a file and to have the shell execute the file, but since the copy/paste scenario is a common use case for us, it seemed worth reporting here.

A secondary observation is that, when any text is pasted into the shell, the text is automatically highlighted (background and foreground colors inverted), as if the user had click+drag-selected the text with the mouse.

**Versions (OS, Maven, Java, and others, as appropriate):**
 - Affected version(s) of this project: 2.1.0
 - OS: CentOS 7.5, RHEL 8

**To Reproduce**
1. Copy/paste valid newline-separated commands into the shell
2. Observe that the command text is automatically highlighted
3. Hit ENTER
4. Observe that none of the commands are executed

**Expected behavior**
The list of commands is executed in order, as with the 1.x shell

**Screenshots**

Commands copied/pasted...

![image](https://user-images.githubusercontent.com/1606834/214347584-66bcca7b-8d27-48b3-b1a4-1c24cb32ec76.png)

After hitting ENTER...

![image](https://user-images.githubusercontent.com/1606834/214348091-1b1403bc-8a63-4712-be4d-38ea7dc01b8b.png)

**Comment by dlmarion** [comment]
I was able to reproduce your issue. IIRC the jline dependency was upgraded and looking at their documentation, issues, and user list, I'm guessing this is the issue. I did notice two things:

 1. When you press a key on the keyboard, the highlighting goes away.
 2. If you paste a single line and hit enter, it works.
 
 So, I think the issue is only with dealing the pasting of multiple lines (and the newline after them).
